thirdparty/vars.sh

```diff
@@ -415,9 +415,27 @@ CLUCENE_NAME="starrocks-clucene-2023.11.28.tar.gz"
 CLUCENE_SOURCE="starrocks-clucene-2023.11.28"
 CLUCENE_MD5SUM="d71f8156bbcbe78e5064ec5616072a5a"
 
+# libunistring
+LIBFUNISTRING_DOWNLOAD="https://ftp.gnu.org/gnu/libunistring/libunistring-1.2.tar.gz"
```

Comment:
this lib is under GPL license, I'm afraid we can't introduce this lib into our repo.